groclient/client.py

```diff
@@ -1716,8 +1716,8 @@ def get_area_weighted_series(self, series_name, weight_names, region_id, method=
         weight_names: list of strs
             List of weight names that will be used to weight the provided series. e.g. ['Barley (ha)', 'Corn (ha)']
             For getting the full list of valid weight names, please call :meth:`~.get_area_weighting_weight_names`
-        region_id: integer
-            The region for which the weighted series will be computed
+        region_id: integer or list of integers
```

Comment:
Rename to region_id`s`? Are there any restrictions, like equal to or above level5?

Reply:
Also, would be nice to add type annotations.

Reply:
yeah I was considering renaming the variable, but I thought if any end user was specifying the parameter like `region_id=1215` then this would be a breaking change. That use case is probably not very likely though (e.g. most users are probably just using positional arguments and not named arguments).

Reply:
as far as restrictions go I'm not entirely sure, I think if you request a region below L5 the endpoint will return an error response, but that's not new behavior

Reply:
yeah so for example, I took Minneapolis (id: ) and ran it against the existing endpoint and this returns a 500 response: 
```
area-weighting?weightNames=Wheat&seriesName=NDVI_8day&regionId=1001236
...
{
    "statusCode": 500,
    "error": "Internal Server Error",
    "message": "An internal server error occurred"
}
```
I can update the comment to make it clear that regions below level 5 are not supported

Reply:
updated with type hints and a comment specifying supported region levels

Reply:
I think we should avoid breaking changes here-- if we rename `region_id` to `region_ids` we'll be breaking any code that currently uses the parameter. 

Agree that it's an inconsistency, but it has precedent in `get_data_points` ([link](https://developers.gro-intelligence.com/api.html#groclient.GroClient.get_data_points))

Reply:
Ah, I didn't realized region_id is a union. Thanks for the update.